Highrachy/ballers-ws
# #172045811 Prepare Ballers for index page

#### What does this PR do?
The diff in this PR does a skeletal setup for the Ballers index page.

#### Description of Task to be completed?
1. Delete the old public folder
2. Update SCSS automation via npm script (in package.json)
3. Create a skeletal index.php file
4. Load Favicon, CSS and jQuery in index.php file

#### How should this be manually tested?
1. Run npm install to install dependencies
2. Visit localhost/ballers/public to view skeletal index file
3. Add sample style to `_index.scss`
4. Ensure that the scss changes is transpiled to css by running `npm run start`. This will run in watch mode and any other changes made to any scss file will be automatically transpiled when the script is still running.
